tests/pytests/integration/modules/test_cp.py

```diff
@@ -0,0 +1,685 @@
+import hashlib
+import logging
+import os
+import pathlib
+import signal
+import textwrap
+import time
+import uuid
+
+import psutil  # pylint: disable=3rd-party-module-not-gated
+import pytest
+import salt.utils.files
+import salt.utils.path
+import salt.utils.platform
+import salt.utils.stringutils
+from saltfactories.utils.ports import get_unused_localhost_port
+from saltfactories.utils.tempfiles import temp_file
+from tests.support.runtests import RUNTIME_VARS
+from tests.support.unit import skipIf
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.slow_test
```

Comment:
```suggestion
log = logging.getLogger(__name__)

pytestmark = [
    pytest.mark.slow_test,
]


```
All `@pytest.mark.slow_test` decorators can now be removed